src/schema/v2/show.ts

```diff
@@ -414,7 +414,7 @@ export const ShowType = new GraphQLObjectType<any, ResolverContext>({
       location: {
         description:
           "Where the show is located (Could also be a fair location)",
-        type: Location.type,
+        type: LocationField.type,
```

Comment:
Export the type separately instead and use that here, because the `type` of a `GraphQLFieldConfig` is a wider union than the exact type that `LocationType` has.

Reply:
Updated to just pass the type here.